packer/.github/workflows/build-ami.yml

```diff
@@ -0,0 +1,11 @@
+name: Build AMI
```

Comment:
Looks like this file wasn't meant to be committed, right?
Also, the path is invalid

Reply:
Right, deleted